lua/lazyvim/plugins/extras/lang/java.lua

```diff
@@ -182,6 +192,10 @@ return {
         callback = function(args)
           local client = vim.lsp.get_client_by_id(args.data.client_id)
           if client and client.name == "jdtls" then
+            -- INFO: without this the inlay hints start toggled off by default.
+            -- Maybe because `nvim-jdtls` attaches on a `FileType` event??
+            -- Still with this the inlay hints are toggled on by default
```

Comment:
```suggestion
            -- INFO: without this the inlay hints start toggled off by default.
            -- Probably because `nvim-lspconfig` is not used to set up `nvim-jdtls`
            -- and `LazyVim.toggle.inlay_hints` doesn't execute, so we have
            -- to execute it again ourselves in this autocmd in `LspAttach`
```

Reply:
I believe this must be a more correct comment as to why we have to call it here again.

Reply:
I've already added your comment changes.